app/library/postgres.py

```diff
@@ -34,15 +34,15 @@ async def post_asset_to_postgres(data: dict, session):
     return asset_model.as_dict()
 
 
-async def get_object_from_postgres(object_id: str, session):
+async def get_object_from_postgres(object_id: str, session, update_cache=False):
```

Comment:
`update_cache` is always `True` regardless of the config value.. I'm guessing that is going to be implemented in another PR?

Reply:
Did you mean always false? and I think this is a default value for if the field isn't set

Reply:
The function that calls `get_object_from_postgres` is always passing in true

Reply:
Ah I see, lines 21-25 in update_cache.py. Good point

Reply:
Nope this is meant to be always true in that function.
The config's `update_enabled` being `True` or `False` determines whether this function runs or not. 
If this function runs, the `update_cache` parameter should always be true, otherwise there is no point in running this function 😄 Does that make sense?

Reply:
An example of where `update_cache` would be `False` is the admin routes. For the user routes, and the update_cache.py we always want it to be true, so that it updates the cache. The logic for the config's `update_enabled` being `True` or `False` is handled in main (https://github.com/uwblueprint/cdc-backend/pull/65/files#diff-a4bc3d28ebe484212cebe359972290c5dd689f559c1dda8d6f71014815f313d1R109)

Reply:
Ahhh got it. So the config's `update_enabled` only affects the admin routes, not the user routes. That makes sense, ty

Reply:
Well no, sorry, to clarify:
- config's `update_enabled` determines whether calling the `update_cache` method periodically is enabled or not.
- the parameter `update_cache` is always False for admin routes, and always True for user routes, independent of config's `update_enabled`

Does that make sense?

Reply:
Oh got it now. The naming tripped me lol

Reply:
Fair enough, I will update the config's name to `periodic_callback_enabled`

Reply:
Updated, please re-approve when you get a chance, ty!